docs/authentication/saml/azure.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Microsoft Azure AD
+description: Learn how to integrate Microsoft Azure AD with Clerk using SAML SSO.
+---
+
+# Microsoft Azure AD
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+    },
+    {
+      title: "Add the appropriate Clerk SDK to your application",
+      link: "/docs/quickstarts/overview",
+    },
+  ]}
+>
+
+- Use Microsoft Azure Active Directory (Azure AD) to enable single sign-on (SSO) via SAML for your Clerk application.
+
+</TutorialHero>
+
+Integrating Microsoft Azure AD with Clerk allows you to use it as an Identity Provider (IdP) for your Clerk application to authenticate your users and manage their access to your application via SAML SSO.
+
+## Tutorial
+
+<Steps>
+
+### Set up Enterprise Connection in Clerk
+
+To create a SAML connection in Clerk:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com).
+2. In the navigation sidebar, select **User & Authentication > [Enterprise Connections](https://dashboard.clerk.com/last-active?path=user-authentication/enterprise-connections)**.
+3. Select the **Create connection** button.
+4. You will be presented with a modal to create a new connection. Fill in the required fields.
+- **Name** - A unique name for the connection. This is so you can tell your connections apart.
+- **Domain** - The email domain for which you will enable Enterprise SSO.
+- **Identity Provider** - The IdP you are using. For this tutorial, select **Microsoft Azure AD**.
+5. Once you have filled in these fields, you will be directed to the **Connection** page which has the configuration settings for the connection you just created. **Do not select the Enable connection toggle just yet.** You need to fully configure your IdP first before exposing your connection to your users while you're in the, sometimes long, process of configuring and testing it.
+6. Leave this page open. You will need to come back to it to complete the setup.
+
+### Create a new Enterprise Application in Azure
+
+To create a new Enterprise Application in Azure:
```

Comment:
out of curiosity, why are we capitalizing "enterprise application"?

Reply:
yeah jacob and I kinda touched on this, but I agree I think we can skip capitalizing. We only capitalize Enterprise Connection in the dashboard